internal/pkg/deploy/cloudformation/task.go

```diff
@@ -48,3 +50,67 @@ func (cf CloudFormation) DeployTask(input *deploy.CreateTaskResourcesInput, opts
 
 	return nil
 }
+
+// ListTaskStacks returns all the CF stacks which represent one-off copilot tasks in a given application's environments
+func (cf CloudFormation) ListTaskStacks(appName, envName string) ([]deploy.TaskStackInfo, error) {
+	taskAppEnvTags := map[string]string{
+		deploy.TaskTagKey: "",
+		deploy.AppTagKey:  appName,
+		deploy.EnvTagKey:  envName,
+	}
+	tasks, err := cf.cfnClient.ListStacksWithTags(taskAppEnvTags)
+
+	if err != nil {
+		return nil, err
+	}
+	var outputTaskStacks []deploy.TaskStackInfo
+	for _, task := range tasks {
+
+		outputTaskStacks = append(outputTaskStacks, deploy.TaskStackInfo{
+			StackName: aws.StringValue(task.StackName),
+			App:       appName,
+			Env:       envName,
+
+			RoleARN: aws.StringValue(task.RoleARN),
+		})
+	}
+	return outputTaskStacks, nil
+}
+
+// ListDefaultTaskStacks returns all the CF stacks created by copilot but not associated with an application.
+func (cf CloudFormation) ListDefaultTaskStacks() ([]deploy.TaskStackInfo, error) {
+	tasks, err := cf.cfnClient.ListStacksWithTags(map[string]string{deploy.TaskTagKey: ""})
+	if err != nil {
+		return nil, err
+	}
+	var outputTaskStacks []deploy.TaskStackInfo
+	for _, task := range tasks {
+		// Eliminate tasks which are tagged for a particular copilot app or env.
+		var hasAppTag, hasEnvTag bool
+		for _, tag := range task.Tags {
+			if aws.StringValue(tag.Key) == deploy.AppTagKey {
+				hasAppTag = true
+			}
+			if aws.StringValue(tag.Key) == deploy.EnvTagKey {
+				hasEnvTag = true
+			}
+		}
+		if hasAppTag || hasEnvTag {
+			continue
+		}
+
+		outputTaskStacks = append(outputTaskStacks, deploy.TaskStackInfo{
+			StackName: aws.StringValue(task.StackName),
+			App:       "",
+			Env:       "",
+
+			RoleARN: "",
+		})
```

Comment:
Maybe for simplicity
```suggestion
		outputTaskStacks = append(outputTaskStacks, deploy.TaskStackInfo{
			StackName: aws.StringValue(task.StackName),
		})
```